api/processes_api.py

```diff
@@ -34,6 +35,10 @@ def do_get(self, **kwargs):
     feature_process = processes.ALL_PROCESSES.get(
       f.feature_type, processes.BLINK_LAUNCH_PROCESS)
 
+    
+    if feature_id != core_enums.FEATURE_TYPE_ENTERPRISE_ID and f.breaking_change:
```

Comment:
Shouldn't this be `if f.feature_type != core_enums.FEATURE_TYPE_ENTERPRISE_ID and f.breaking_change:` (change feature_id to f.feature_type)

Reply:
Can you add another test case to https://github.com/GoogleChrome/chromium-dashboard/blob/main/api/processes_api_test.py that checks this branch?

Reply:
> 

We actually want to insert the feature rollout stage in non enterprise features if they are breaking, we do not want to change the feature type altogether.

Reply:
I misunderstood. Fixed it.